src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -2677,6 +2693,18 @@ namespace Akka.Dispatch
         public RejectedExecutionException(string message = null, System.Exception inner = null) { }
         protected RejectedExecutionException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
     }
+    public enum TaskSchedulerPriority
```

Comment:
Is there a built-in enum we can leverage instead of adding our own? Or is that all "thread priority" stuff and not "task priority" ?

Reply:
Yes, the values are the same like on windows for thread and process priority
I did not find any good inside .net, in worst case they can be casted, when ever needed.

Reply:
I added a comment